requirements.txt

```diff
@@ -29,4 +29,4 @@ pybind/pybind11@d159a563383d10c821ba7b2a71905d1207db6de4 --build
 msgpack/msgpack-c@cpp-3.3.0 -DMSGPACK_BUILD_TESTS=Off
 sqlite3@3.43.2 -DCMAKE_POSITION_INDEPENDENT_CODE=On
 ROCmSoftwarePlatform/composable_kernel@70eefcf4f263aa5c25f3c9ff0db8f6f199ef0fb9 -DCK_BUILD_JIT_LIB=On -DCMAKE_POSITION_INDEPENDENT_CODE=On
-ROCmSoftwarePlatform/rocMLIR@9e66e8050209f03349a41b6b497f0da2b285a53b -DBUILD_FAT_LIBROCKCOMPILER=On
+ROCmSoftwarePlatform/rocMLIR@5085343bca363109ae9ebabb7ca2b65c52bc861c -DBUILD_FAT_LIBROCKCOMPILER=On
```

Comment:
Does this hash make sense still?  There have been an update since this work started

Reply:
I need to update this hash.

Reply:
The tip of our `develop` has all the fp8 fixes and your PR

Reply:
Reverted this change. SHA from develop branch works well now.